developer/src/kmc-copy/src/KeymanProjectCopier.ts

```diff
@@ -687,4 +696,11 @@ export class KeymanProjectCopier implements KeymanCompiler {
     return true;
   }
   /* c8 ignore stop */
+
+  /** @internal */
+  public unitTestEndPoints = {
+    getGithubSourceProject: this.getGitHubSourceProject.bind(this),
+    getCloudSourceProject: this.getCloudSourceProject.bind(this)
+  };
```

Comment:
This pattern allows us to expose private functions for unit testing without breaking encapsulation. We use a similar pattern for global functions in a module, where we don't want to export them directly.

@markcsinclair fyi.

https://github.com/keymanapp/keyman/wiki/Unit-Tests#typescript